wolfssl/internal.h

```diff
@@ -1989,6 +1989,15 @@ enum Misc {
     #define MAX_CHAIN_DEPTH 9
 #endif
 
+/* Max certificate extensions in TLS1.3 */
+#if defined(HAVE_CERTIFICATE_STATUS_REQUEST)
+    /* Number of extensions to set each OCSP response */
+    #define MAX_CERT_EXTENSIONS (1 + MAX_CHAIN_DEPTH)
```

Comment:
Should be overridable if is an arbitrary number chosen?

Reply:
I think we could consider that `MAX_CERT_EXTENSIONS` is non overridable because it represents the maximum number of extensions which is determined by `MAX_CHAIN_DEPTH`.That is the case, you think `MAX_CERT_EXTENSIONS` should be enumeration rather than macro so that it is able to be non overridable?